ReClass.NET/Controls/MemoryViewControl.cs

```diff
@@ -704,5 +704,24 @@ public void Reset()
 
 			VerticalScroll.Value = VerticalScroll.Minimum;
 		}
+
+
```

Comment:
Could you please remove these double empty lines here and in the other files?

Reply:
Done